tests/spec/core/examples-spec.js

```diff
@@ -39,10 +39,12 @@ describe("Core — Examples", () => {
     expect(markers.length).toBe(1);
 
     const [marker] = markers;
-    expect(marker.textContent).toBe("Example 1: EX");
+    expect(marker.textContent.trim()).toBe("Example 1: EX");
     expect(marker.querySelector(".example-title").textContent).toBe(": EX");
     expect(example.getAttribute("title")).toBeNull();
-    expect(example.textContent).toBe("Example 1: EX\n{\n  CONTENT\n}\n  ");
+    expect(example.textContent.trim()).toBe(
+      "Example 1: EX\n    \n{\n  CONTENT\n}"
```

Comment:
Maybe we should normalize this and remove the spaces... we don't really care about the \n

Reply:
(I guess if works for now, it's fine... otherwise, we will need to normalize the others)

Reply:
I'll merge this first and do it on another PR.